### PR TITLE
Additional unit tests

### PR DESCRIPTION
**Pull request by scottdixon**

### CODE REVIEW

**Comment by scottdixon** [comment]
@moizjv @imurchie @jlipps

File: package.json [hunk 1]
```diff
@@ -60,7 +60,8 @@
     "chai-as-promised": "^5.1.0",
     "gulp": "^3.8.11",
     "mochawait": "^2.0.0",
```
Comment (imurchie) [inline package.json]: `mochawait` is no longer necessary.

File: test/functional/android-helper-e2e-specs.js [hunk 2]
```diff
@@ -11,26 +7,18 @@ import ADB from 'appium-adb';
...
 
-/*const should = */chai.should();
+chai.should();
 chai.use(chaiAsPromised);
 
 describe('android-helpers e2e', () => {
-
   describe('installApkRemotely', () => {
-
     it.skip('installs an apk by pushing it to the device then installing it from within', async function () {
```
Comment (jlipps) [inline test/functional/android-helper-e2e-specs.js]: if we want to keep this skip in committed code, we might want an explanation as to why

**Comment by jlipps** [comment]
:+1:  looking good!

**Pull request merged**